source/includes/cpp-library-reference/_simulator.html.md

```diff
@@ -174,10 +155,10 @@ The default implementation will throw an exception.
 
 > Example Inkling:
 
-```inkling
-schema Action
-    Int8{0, 1} delta
-end
+```inkling2
+type Action {
+    delta: Number.Int8<0, 1>
```

Comment:
Technically, I think this needs to use named values to preserve the previous behavior of using DQN in the backend, right? e.g. `Number.Int8<Left=0, Right=1>`.